pyproject.toml

```diff
@@ -60,13 +61,17 @@ license-files = ["LICENSE"]
 [tool.setuptools.dynamic]
 dependencies = {file = "requirements/core.txt"}
 readme = {file = "README.rst", content-type = "text/x-rst"}
-version = {attr = "iris_grib.__version__"}
 optional-dependencies.all = {file = "requirements/all.txt"}
 optional-dependencies.test = {file = "requirements/test.txt"}
 
 [tool.setuptools.packages.find]
 include = ["iris_grib*"]
 
+[tool.setuptools_scm]
+write_to = "iris_grib/_version.py"
```

Comment:
You might have noticed that this new file is polluting your `git status`. We need an addition to the `.gitignore` file, like [this one in Iris](https://github.com/SciTools/iris/blob/7a3248668d57c7e6010dd8a9c4ee8602124f1bf9/.gitignore#L3-L4).